VideoLocker/res/values/strings.xml

```diff
@@ -220,6 +220,8 @@
     <string name="no_information_to_display">Additional course information is currently unavailable.</string>
 
     <!-- Assessment -->
+    <!-- Assessment screen name, for event used in screenViewsTracking-->
+    <string name="assessment_screen">Assessment</string>
```

Comment:
Also, it looks like you're using it on one of the navigation screens? Redoing this is part of the assessments project, but it's still just the course outline screen?